pkg/storaged/storage.css

```diff
@@ -641,3 +641,71 @@ td.storage-action {
     opacity: 0;
     visibility: hidden;
 }
+
+.modal-body .intermission {
+    margin-top: 1.5ex;
+    margin-bottom: 1ex;
+    white-space: normal;
+    line-height: 1.6666;
+    color: rgb(54, 54, 54);
+}
+
+.modal-body .widest-title {
+    visibility: hidden;
+    height: 0px;
+}
+
+span.radio input[type=radio] {
+    margin-top: 5px;
+}
+
+span.radio label {
+    margin-right: 1em;
+}
+
+.key-slot-panel-remaining {
+    margin-right: 1em;
+}
+
+.key-slot-panel td:nth-child(1),
+.key-slot-panel td:nth-child(3),
+.key-slot-panel td:nth-child(4) {
+    width: 1%;
+    white-space: nowrap;
+}
+
+.key-slot-panel td:nth-child(1) {
+    padding-right: 1em;
+}
+
+.key-slot-panel td:nth-child(3) {
+    padding-right: 2em;
+}
+
+.sigkey-hash {
+    font-family: monospace;
+    font-size: 200%;
+}
+
+pre.inline-pre {
+    display: inline;
+    padding: 4px;
+}
+
+.pficon.large {
+    font-size: 200%;
+}
+
+.large {
+    font-size: 150%;
+}
+
+#dialog input.radio-child {
+    margin-left: 20px;
+    width: 450px;
+}
+
+#dialog div.radio-child {
```

Comment:
1. It's better to not use IDs in CSS, when possible.
2. Why is there a `.radio-child` that's on both an input and a div?
3. Why is the width set to pixels in the input?
4. Why is color red set in the div? We should use a [PatternFly color](https://www.patternfly.org/styles/color-palette/) (this is "pf-red-200", which is #a30000)  instead of a generic named CSS color.
5. Why aren't the shared properties grouped?

---

I guess most of this stems from the passphrase entry versus slot when removing the dialog. Just from the CSS, I wouldn't be able to figure that out. I'd recommend using unique classes on each element and have the names be descriptive.

Suggested names (based on BEM, which PatternFly 4 will use):

`.nbde__dialog--remove__passphrase__entry`
`.nbde__dialog--remove__slot__warning`

As we haven't decided to actually adopt BEM (or something like it) for ourselves (yet?), something descriptive like this would also suffice:

`.nbde-dialog-remove-passphrase-entry`
`.nbde-dialog-remove-slot-warning`

_or_ we could use 1 level of the cascade:

`.nbde-dialog .passphrase-entry`
`.nbde-dialog .slot-warning`

(I personally like the last best.)

Still, it would probably be better to make both generic for the indentation. If indentation could be added by nesting within a parent element (the widget that toggles the disclosure or a container around it), all the better. I've looked for an example on PatternFly.org, and code examples for how to best do progressive disclosure do not seem to exist, however.

Reply:
Yes, that was a terrible choice of names.

Reply:
Okay, it looks like this now:
```
.remove-passphrase .indented {
    padding-left: 20px;
    width: 100%;
}

.remove-passphrase .slot-warning {
    color: #a30000;
}
```

The 450px was there so that the input field didn't stick out the dialog.  Now it is nested inside a padded div and we can let it keep it's natural 100% width.

The storage page still uses plain CSS.  Should we move to less? or sass?  Then we can use grouping and variables.  (Or can we use them with CSS now?)